src/App/Fossa/Analyze.hs

```diff
@@ -257,26 +277,43 @@ analyze (BaseDir basedir) destination override unpackArchives filters = do
         -- Warn on contributor errors, never fail
         void . Diag.recover . runExecIO $ tryUploadContributors basedir apiOpts (uploadLocator uploadResult)
 
+
 data CountedResult
   = NoneDiscovered
   | FilteredAll Int
-  | FoundSome (NE.NonEmpty ProjectResult)
+  | ManualOnly SourceUnit
+  | FoundSome (NE.NonEmpty SourceUnit)
```

Comment:
I'm not sure I understand why we have two different types here. I think separating `FoundSome` and `ManualOnly` adds a bit of confusion as well as the necessity to handle both cases near line 237. In my opinion this would look cleaner if we only used `FoundSome`.

Reply:
I think you're right, this was necessary for a different attempt at merging the file with normal analysis.  I can revisit to check.